web/app/public/index.html

```diff
@@ -8,10 +8,17 @@
     <title>{{ .Title }}</title>
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
     <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png" />
     <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png" />
+    <link rel="manifest" href="/manifest.json">
+    <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
```

Comment:
You can call me petty, but after doing a quick search and seeing that only safari supports `mask-icon`, I would personally rather not have line and keep things minimal. 

Don't blame me, blame Safari refusing to implement things that every other browsers support (svg favicons using `rel="icon"`).

Reply:
Agree with you, removed this line